src/ducks/notifications/CategoryBudget/index.js

```diff
@@ -68,7 +68,7 @@ const transformForTemplate = (budgetAlert, t, accountsById, groupsById) => {
   return {
     ...budgetAlert.alert,
     categoryLabel: t(`Data.${type}.${catName}`),
-    currentAmount: (-sumBy(budgetAlert.expenses, tr => tr.amount)).toFixed(0),
+    currentAmount: Math.round(-sumBy(budgetAlert.expenses, tr => tr.amount)),
```

Comment:
Why are we switching from `toFixed()` to `Math.round()`? Can you explain it in the commit message? And maybe create a specific commit for this change? 

I know that for positive number, round() and toFixed() should return the same value and that round() is a lot more faster that toFixed, but is it really why we changed it?

Reply:
You can start adding jsdoc to the app (with a ts-check tag) (or adding ts file) to help you during the dev process